code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -202,28 +202,19 @@ var/bomb_set
 		ticker.mode:syndies_didnt_escape = (Shuttle && Shuttle.z == ZLEVEL_CENTCOM) ? 0 : 1
 		ticker.mode:nuke_off_station = off_station
 	ticker.station_explosion_cinematic(off_station,null)
+
+	ticker.mode.station_was_nuked = (off_station<2)	//offstation==1 is a draw. the station becomes irradiated and needs to be evacuated.
+													//kinda shit but I couldn't  get permission to do what I wanted to do.

```

Comment:
Should probably axe the comment then, since it's likely nobody knows what the hell this guy was talking about.